_integration-test/conftest.py

```diff
@@ -10,27 +10,49 @@
 TEST_PASS = "test123TEST"
 TIMEOUT_SECONDS = 60
 
+def pytest_addoption(parser):
+    parser.addoption("--customizations",  default=False)
 
 @pytest.fixture(scope="session", autouse=True)
-def configure_self_hosted_environment():
+def configure_self_hosted_environment(request):
     subprocess.run(["docker", "compose", "--ansi", "never", "up", "-d"], check=True)
     for i in range(TIMEOUT_SECONDS):
         try:
             response = httpx.get(SENTRY_TEST_HOST, follow_redirects=True)
-        except httpx.ConnectionError:
+        except httpx.NetworkError:
             time.sleep(1)
         else:
             if response.status_code == 200:
                 break
     else:
         raise AssertionError("timeout waiting for self-hosted to come up")
 
+    if request.config.getoption("--customizations") == "True":
+        os.environ['TEST_CUSTOMIZATIONS'] = "True"
+        script_content = '''\
```

Comment:
These lines are definitely covered by testing, not sure why Codecov is complaining here.

Reply:
That's the reason why `Test / integration test (True, v2.0.1)` and `Test / integration test (True, v2.7.0)` are taking 8 minutes longer

Reply:
nit: It's pretty hard for a naive observer to figure out what the "True" in these variants means. Maybe we could have values be `Customization Enabled` vs `Customization Disabled` rather than `True`/`False`?

Reply:
Apparently, you can create a custom name for the job. Now it looks like
`Test / integration test v2.0.1 - customizations disabled`